src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/boss_salramm_the_fleshcrafter.cpp

```diff
@@ -57,7 +57,11 @@ class boss_salramm : public CreatureScript
         {
             boss_salrammAI(Creature* creature) : BossAI(creature, DATA_SALRAMM) { }
 
-            void InitializeAI() override { Talk(SAY_SPAWN); }
+            void InitializeAI() override {
```

Comment:
Yeah TC code style mandates newline if it's not a one-liner. I just missed it when adjusting.

You're commenting on an oudated diff though.